src/lib/helper.py

```diff
@@ -79,19 +79,19 @@ def pi_fractions(val, pi='pi') -> str:
     return ''
   if val == 0:
     return '0'
-  for pi_multiplier in range(1, 2):
-    for frac in range(-128, 128):
-      if frac and math.isclose(val, pi_multiplier * math.pi / frac):
+  for pi_multiplier in range(1, 128):
```

Comment:
This 128 strikes as a bit high. For larger circuits with perhaps 100,000 or more gates, this will become a transpiler performance problems. Suggest to keep it around 2, perhaps 4 max.

Reply:
Done.